src/reducer/database/models.ts

```diff
@@ -0,0 +1,1066 @@
+    fields.repositoryId = this.id;
+
+    let language = Language.objects.create(fields);
+
+    return language as Language;
+  }
+
+  /**
+   * Get all members within this repository.
+   *
+   * @returns {Member[]} A list of members object
+   */
+  getMembers(): Member[] {
+    let members = RepositoryHasMember.objects.filter(
+      {
+        repositoryId: this.id,
+      },
+      Member
+    );
+
+    return members as Member[];
+  }
+
+  /**
+   * Get the analysis of all language data of this repository.
+   *
+   * @returns {Language[]} A language evaluation object
+   */
+  getLanguages(): Language[] {
+    let response = Language.objects.custom(
+      Language.statements.byRepository(this.id)
+    ) as Language[];
+
+    return response;
+  }
+
+  /**
+   * @todo Implement a save functionality
+   * @description Saves this osm model
+   */
+  save() {}
+}
+
+/**
+ * @class A OSM model for a repositoryHasMember object
+ * @extends osm.models.RepositoryHasMemberSO Statement object
+ */
+class RepositoryHasMember extends osm.models.RepositoryHasMemberSO {
+  public static objects = osm.models.RepositoryHasMemberSO.getObjects(
+    RepositoryHasMember
+  );
+
+  public objects = RepositoryHasMember.objects;
+  public id = 0;
+
+  /**
+   * @todo Implement a save functionality
+   * @description Saves this osm model
+   */
+  save() {}
+}
+
+/**
+ * @class A OSM model for a language object
+ * @extends osm.models.LanguageSO Statement object
+ */
+class Language extends osm.models.LanguageSO implements ILanguage {
+  public static objects = osm.models.LanguageSO.getObjects(Language);
+  public objects = Language.objects;
+
+  public id = 0;
+  public color = "";
+  public name = "";
+  public size = 0;
+  public share = 0;
+  public repositoryId = 0;
+
+  constructor(args: ILanguage) {
+    super();
+
+    this.id = args["id"];
+    this.color = args["color"];
+    this.name = args["name"];
+    this.size = args["size"];
+    this.share = args["share"];
+    this.repositoryId = args["repositoryId"];
+  }
+
+  /**
+   * Get general language evaluation.
+   *
+   * @returns {Language[]} A list of language objects
+   * @description Get a language analysis of all language data
+   */
+  static getLanguages(): Language[] {
+    let response = this.objects.custom(
+      Language.statements.merged
+    ) as Language[];
+
+    response = response.map((entry) => {
+      return new Language(entry);
+    });
+
+    return response;
+  }
+
+  /**
+   * @todo Implement a save functionality
+   * @description Saves this osm model
+   */
+  save() {}
+}
+
+/**
+ * @class A OSM model for a platformHasRepository object
+ * @extends osm.models.PlatformHasRepositorySO Statement object
+ */
+class PlatformHasRepository extends osm.models.PlatformHasRepositorySO {
+  public static objects = osm.models.PlatformHasRepositorySO.getObjects(
+    PlatformHasRepository
+  );
+
+  public id = 0;
+  public objects = PlatformHasRepository.objects;
```

Comment:
A newline has to be added, because this is a logical break and the other classes also had a newline between the id and objects.
```suggestion
  public objects = PlatformHasRepository.objects;

  public id = 0;
```